bundle/config/mutator/python/python_mutator.go

```diff
@@ -261,10 +263,21 @@ func (m *pythonMutator) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagno
 			panic("mutateDiags has no error, but error is expected")
```

Comment:
Let's not have panics in mutators; just return regular error.

```
   if err == mutateDiagsHasError {
		if !mutateDiags.HasError() {
```

This error handling seems unnecessary complicated. Why not:
 - accumulate diags properly inside b.Config.Mutate
 - checks accumulated diags for errors after the function and return them.
 - not use "sentinel" error.

Reply:
Ok. This can be possibly improved, but this is not relevant to this PR, right?

Reply:
Well you build on top of it below, which makes error handling even harder to read. Can be follow up - up to you.

Reply:
I agree we should fix, but let's do it separately